Sources/xcproj/XCWorkspace.swift

```diff
@@ -34,6 +34,12 @@ public class XCWorkspace {
         self.data = data
     }
 
+    /// Returns an empty workspace.
+    public static var `default`: XCWorkspace {
```

Comment:
This makes it seem as if it's a singleton. You could remove this and just make a new initializer that doesn't take `data`.

Reply:
Yeah, makes more sense. Let me update it.

Reply:
Done @yonaskolb